src/components/Avatar/Avatar.test-int.js

```diff
@@ -0,0 +1,104 @@
+/* global Feature, Scenario */
+const searchWithinIframe = require('../../../test/codecept/utils/searchWithinIframe.js');
+
+const KIND = 'Avatar.Avatar';
+
+Feature('Avatar');
+
+Scenario('Overview', (I) => {
+    searchWithinIframe(I, KIND, 'Overview', () => {
+        I.see('Avatar: Avatar');
+        I.see('Overview');
+        I.see('Example');
+        I.seeElement('.uir-avatar');
+        I.see('DS');
+        I.see('Source Code');
+        I.seeElement('code');
+        I.see('Props');
+        I.seeElement('table');
+        I.see('property');
+        I.see('value');
+        I.see('required');
+        I.see('default');
+        I.seeNumberOfElements('td', 12 * 4);
```

Comment:
Again, AFAIK the props table is something that Storybook does out of the box, so I'm not sure if we need to test it?

Reply:
We've built a custom props table plugin (which is unit tested)... I'll bow to your expertise over whether it should be included in the scope of the int tests

My thought process was it's important for users to be able to make use of the documentation so we should check that it is being displayed...

Reply:
All integration tests have been refactored to make them simpler, reduce duplication and run faster as a result

Reply:
I agree that we should test the rendering of the docs too. I think what you've done is a good balance :100: